pycolmap/geometry/bindings.h

```diff
@@ -30,9 +31,10 @@ void BindGeometry(py::module& m) {
       .def("inverse", &Eigen::Quaterniond::inverse)
       .def("__repr__", [](const Eigen::Quaterniond& self) {
         std::stringstream ss;
-        ss << "Rotation3d: " << self.coeffs();
+        ss << "Rotation3d(quat_xyzw=[" << self.coeffs().transpose() << "])";
         return ss.str();
       });
+  py::implicitly_convertible<py::array, Eigen::Quaterniond>();
```

Comment:
That's a nice idea, maybe also add a conversion from a 3x4 matrix directly to Sim3d / Rigid3d?